spec/PeriodicBackgroundSync-index.bs

```diff
@@ -0,0 +1,95 @@
+
+  As this API relies on service workers, functionality provided by this API is only available in a [=secure context=].
+  ## Example ## {#example}
+  Requesting a [=periodic Background Sync opportunity=] at a daily frequency from a [=browsing context=]:
+
+  <pre class="lang-js">
+    async function getLatestNews() {
+      const registration = await navigator.serviceWorker.ready;
+      try {
+        await registration.periodicSync.register('get-latest-news', {
+          minInterval: 24 * 60 * 60 * 1000,
+        });
+      } catch (error) {
+        console.log('Periodic Sync could not be registered!');
+      }
+    }
+  </pre>
+
+  Reacting to a periodicsync event within a [=service worker=]:
+
+  <pre class="lang-js">
+    self.addEventListener('periodicsync', event => {
+      event.waitUntil(fetchAndCacheLatestNews());
+    });
+  </pre>
+
+  In the above example <code>fetchAndCacheLatestNews</code> is a developer-defined function.
+
+Concepts {#concepts}
+========================
+
+The periodicsync event is considered to run <dfn>in the background</dfn> if no [=service worker clients=] whose [=frame type=] is top-level or auxiliary exist for the origin of the corresponding service worker registration.
+
+The user agent is considered to be <dfn>online</dfn> if the user agent has established a network connection. A user agent MAY use a stricter definition of being [=online=]. Such a stricter definition MAY take into account the particular [=service worker=] or origin a [=periodicsync registration=] is associated with.
+
+A <dfn>periodic Background Sync opportunity</dfn> allows periodic synchronization between the server and the web app, the exact frequency of which is decided by the user agent. It can be requested via a [=periodicsync registration=].
+
+Constructs {#constructs}
+=========================
+
+A [=service worker registration=] has an associated <dfn>list of periodicsync registrations</dfn> whose element type is a [=periodicsync registration=].
+
+A <dfn>periodicsync registration</dfn> is a tuple consisting of a [=tag=], a dictionary of options, and a [=registration state=].
+
+The <dfn>tag</dfn> is a DOMString. Within one [=list of periodicsync registrations=] each [=periodicsync registration=] MUST have a unique [=tag=].
```

Comment:
Are all these things (incl. `DOMString`) styled and linkified correctly?

Reply:
I've linked to DOMString.